bofire/data_models/strategies/enting.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Dict, Literal, Type
+
+from bofire.data_models.constraints.api import (
+    Constraint,
+    LinearEqualityConstraint,
+    LinearInequalityConstraint,
+)
+from bofire.data_models.features.api import (
+    CategoricalDescriptorInput,
+    CategoricalInput,
+    ContinuousInput,
+    ContinuousOutput,
+    DiscreteInput,
+    Feature,
+)
+from bofire.data_models.objectives.api import MinimizeObjective, Objective
+from bofire.data_models.strategies.strategy import Strategy
+
+
+class EntingStrategy(Strategy):
+    type: Literal["EntingStrategy"] = "EntingStrategy"
+    enting_params: Dict[str, Any] = {}
+    solver_params: Dict[str, Any] = {}
+
+    @classmethod
+    def is_constraint_implemented(cls, my_type: Type[Constraint]) -> bool:
+        return my_type in [LinearEqualityConstraint, LinearInequalityConstraint]
+
+    @classmethod
+    def is_feature_implemented(cls, my_type: Type[Feature]) -> bool:
+        return my_type in [
+            CategoricalInput,
+            DiscreteInput,
+            CategoricalDescriptorInput,
+            ContinuousInput,
+            ContinuousOutput,
+        ]
+
+    @classmethod
+    def is_objective_implemented(cls, my_type: Type[Objective]) -> bool:
```

Comment:
ENTMOOT doesn't natively support maximization, but I have multiplied the values in the `fit` and `predict` function by `-1`. I've used the Ackley benchmark to demonstrate this (although I'm not sure why this uses `MaximizeObjective`, since Ackley looks to be a minimization problem as far as I can tell).